astroquery/jplhorizons/core.py

```diff
@@ -584,6 +589,14 @@ def ephemerides_async(self, airmass_lessthan=99,
         else:
             request_payload['SKIP_DAYLT'] = 'NO'
 
+        # inject optional settings if provided
+        if optional_settings:
+            assert isinstance(
```

Comment:
asserts shouldn't really be in code, could you please rephrase? E.g. the signature implies that `optional_settings` should be dicts, so duct typing should be fine.

Also, we don't enforce short lines, all this would be more readable with linebreaks at logical separations rather than at any parens.